CoverMyDotNet/Models/Common/RequestResponseAttributes.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using RestSharp.Serializers;
+
+namespace CoverMyDotNet
+{
+	//this class is for the value returned from fetching a request
+	public class RequestResponseAttributes
```

Comment:
@twyfordr Yes, I was going to use two classes for the request and response, but I decided to use just one.